packages/pricing/src/services/pricing-module.ts

```diff
@@ -110,6 +131,17 @@ export default class PricingModuleService<
     const quantity = context.quantity
     delete context.quantity
 
+    // Currency code here is a required param.
+    const currencyCode = context.currency_code
+    delete context.currency_code
+
+    if (!currencyCode) {
+      throw new MedusaError(
+        MedusaError.Types.INVALID_DATA,
+        `currency_code is a required input in the pricing context`
+      )
+    }
+
     // Gets all the price set money amounts where rules match for each of the contexts
     // that the price set is configured for
     const psmaSubQueryKnex = knex({
```

Comment:
Yes, I agree i have this in my todo as well. This will be completed this week in a separate PR since there will be some deviations (or new) from convention that we have.